filters/apiusagemonitoring/pathinfo.go

```diff
@@ -6,13 +6,22 @@ import (
 	"regexp"
 )
 
+type Regex struct{ *regexp.Regexp }
```

Comment:
small consistency nitpick: it's called regexp everywhere in the stdlib and the app, let's keep that 'p'

Reply:
Was also wondering if I should call it something like `JsonRegexp` or `JSRegexp` or something more specifically linked to why it is being extended.

Reply:
following the rule of 'less is more', better keep it just regexp. Oh! Is this type referenced anywhere outside of the filter? Because if not, then we don't need to export it (meaning no need for the capital 'R')

Reply:
![image](https://user-images.githubusercontent.com/39987010/49863155-f5600a80-fdff-11e8-990f-22ba6c820336.png)

Mmmm ... well then that makes a name clash with the import of the `regexp` package...

Reply:
"less is more", it could be then:

```
type rexp struct { *regexp.Regexp }
```